recipe/meta.yaml

```diff
@@ -46,6 +46,7 @@ test:
     {% endfor %}
     - test -f ${PREFIX}/include/bzlib.h  # [unix]
     - test -f ${PREFIX}/lib/libbz2.a  # [unix]
+    - test -f ${PREFIX}/lib/libbz2.so.1.0  # [unix]
```

Comment:
Thanks @marcelm.

Though I think this will create `dylib` extension on Mac. So we will need to do things differently there. Also I believe on Mac we would have the extension last so `libbz2.1.0.dylib`. Maybe can add a branch and check `uname` for `"Linux"` or `"Darwin"`. Symlinking otherwise would be the same.